dev-infrastructure/configurations/svc-cluster.bicepparam

```diff
@@ -31,11 +31,9 @@ param serviceKeyVaultName = take('service-kv-${uniqueString(currentUserId)}', 24
 param serviceKeyVaultSoftDelete = false
 param serviceKeyVaultPrivate = false
 
-param dnsZone = {
-  name: take('dev-${uniqueString(currentUserId)}', 10)
-  parentZoneName: 'hcp.osadev.cloud'
-  parentZoneResourceGroup: 'global'
-}
+param regionalDNSSubdomain = take('dev-${uniqueString(currentUserId)}', 10)
```

Comment:
shoudl we just hardcode this off of the actual regional name, scoped down to 10 chars?